x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/entity_store/entity_store_data_client.ts

```diff
@@ -498,7 +498,6 @@ export class EntityStoreDataClient {
         status: ENGINE_STATUS.ERROR,
         error: {
           message: err.message,
-          stack: err.stack,
```

Comment:
This may be left over from dev not sire, but I don't think we should be saving stacks it feels wrong 🤔